themes/dark/templates/editShow.mako

```diff
@@ -313,31 +309,29 @@ var startVue = function() {
                     }
                 };
             },
-            onChangeIgnoredWords: function(items) {
-		        console.log('Event from child component emitted', items);
-                this.seriesObj.config.release.ignoredWords = items.map(item => item.value);
+            onChangeIgnoredWords(items) {
+                this.series.config.release.ignoredWords = items.map(item => item.value);
             },
-            onChangeRequiredWords: function(items) {
-		        console.log('Event from child component emitted', items);
-                this.seriesObj.config.release.requiredWords = items.map(item => item.value);
+            onChangeRequiredWords(items) {
+                this.series.config.release.requiredWords = items.map(item => item.value);
             },
-            transformToIndexedObject: arrayOfStrings => {
-                arrayOfObjects = [];
-                for (let index=0; index < arrayOfStrings.length; index++) {
-                    arrayOfObjects.push({id: index, value: arrayOfStrings[index]});
-                }
-                return arrayOfObjects;
-            }
+            transformToIndexedObject: (arrayOfStrings) => (arrayOfStrings || []).map((string, index) => ({id: index, value: string}))
         },
         computed: {
-            availableLanguages: function() {
-                if (this.config.indexers.config.main.validLanguages){
-                    return this.config.indexers.config.main.validLanguages.join(',');
+            availableLanguages() {
+                const {validLanguages} = this.config.indexers.config.main;
+                if (validLanguages){
+                    return validLanguages.join(',');
                 }
+            },
+            ignoredWords() {
+                return this.transformToIndexedObject(this.series.config.release.ignoredWords);
+            },
+            requiredWords() {
+                return this.transformToIndexedObject(this.series.config.release.requiredWords);
```

Comment:
@p0psicles Instead of mutating the value in the DOM you should be doing in a computed property.